src/conditions/index.jsx

```diff
@@ -1,15 +1,14 @@
 import React, { Component, Fragment } from 'react';
 import ReactMarkdown from 'react-markdown';
 import { Form } from '../';
-import { Warning } from '@ukhomeoffice/react-components';
 
 class Conditions extends Component {
 
   componentDidMount() {
     this.setState({ editing: false });
   }
 
-  toggleEdit = e => {
+  toggleEdit (e) {
```

Comment:
eslint can't handle the arrow functions because it's not _actually_ legit JS syntax to set class properties like this - it's just handled by babel.